pylua/test/test_timer.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+
+
+import timeit
+import threading
+import pylua.timer as timer
+
+
+class TestPyTimer:
+
+    def _timer_task(self):
+        self._task_is_run = True
+        self._event.set()
+
+    def __init__(self):
+        self._interval = 50  # interval in milliseconds
+        self._timer = timer.PyTimer(self._timer_task, self._interval)
+        self._task_is_run = False
+        self._event = threading.Event()
+
+    def _run_event(self, wait_time):
```

Comment:
@dtrunov, propose to rename `_run_event` -> `_wait_event` since you're not running anything here.